docs/guide/convert_doc.md

```diff
@@ -103,3 +103,55 @@ Simply pass a python list object containing multiple urls. Let `URL` be a list c
     # Let's download all the converted documents locally in RESULT_DIR
     documents.download_all(result_dir = RESULT_DIR)
     ```
+
+---
+
+## Downloading converted documents
+
+- For CLI:
+The converted documents are downloaded automatically.
```

Comment:
Please add a comment about where they could be found. Like a folder with this name "abc".